src/compas_fab/backends/kinematics/client.py

```diff
@@ -0,0 +1,15 @@
+import logging
+from compas_fab.backends import PyBulletClient
+from compas_fab.backends.pybullet import LOG
+from compas_fab.backends.kinematics import AnalyticalInverseKinematics
+from compas_fab.backends.kinematics import AnalyticalPlanCartesianMotion
+
+LOG.setLevel(logging.ERROR)
```

Comment:
Setting this at this level might have a bit unclear effects, because it's only applied when the module is loaded. It's not really a big issue, but just a bit awkward.